File: Modul-2/Soal-5/PRAK205-2310817110013-Randy_Febrian.c
#include <stdio.h>
#include <math.h>

int main(){
    int tinggi, miring;
    scanf("%d %d", &tinggi, &miring);
    int alas = sqrt((miring * miring) - (tinggi * tinggi));
    int keliling = miring + tinggi + alas;
    int luas = (alas * tinggi) / 2;
    
    printf("Alas = %d cm\n", alas);
    printf("Tinggi = %d cm\n", tinggi);
    printf("Keliling = %d cm\n", keliling);
    printf("Luas = %d cm^2\n", luas);
    return 0;
}

File: Modul-2/Soal-5/PRAK205-2310817110013-Randy_Febrian.py
from math import sqrt

jumlah_input = input().split()

# A adalah tinggi dan B adalah miring
if len(jumlah_input) == 2:
    A, B = map(float, jumlah_input)
elif len(jumlah_input) == 1:
    A = float(jumlah_input[0])
    B = float(input())
else:
    print("Masukkan 2 angka secara vertikal atau horizontal")

# C adalah alas
C = sqrt((B * B) - (A * A))
luas = (C * A)/2
keliling = A + B + C

print("Alas = %d cm" % C)
print("Tinggi = %d cm" % A)
print("Keliling = %d cm" % keliling)
print("Luas = %d cm^2" % luas)